packages/lit-dev-content/site/docs/components/properties.md

```diff
@@ -405,6 +516,24 @@ set prop(val: number) {
 get prop() { return this._prop; }
 ```
 
+```js
+static properties = {
+  prop: {},
+};
+
+_prop = 0;
```

Comment:
This example is the first time in the examples where the member initialization isn't in the constructor.  Am I correct in my understanding that it's because this `_prop` isn't in the `static properties` object?

Reply:
Yep, only reactive properties need to be initialized in the constructor.

Reactive properties need to be initialized in the constructor instead of declared as a class field because otherwise the property will be directly defined on the instance, shadowing the automatically-generated setter on the prototype, thus breaking reactivity.

I could certainly imagine it could be confusing to show examples of class fields like this, as opposed to the simpler message "always initialize Lit properties in the constructor", but I think it's the most correct approach since the "reactive" distinction is the key thing.